### PR TITLE
Added pattern randomiser

### PR DESCRIPTION
**Pull request by ryanleewatts**

### CODE REVIEW

File: css/randomiser.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    border-bottom: 1pt solid #fff;
+}
+
+#wheel_meaning {
+    margin-top: 2vh;
+}
+
+#end_card {
+    display: none;
+}
```
Comment (danielleos) [inline css/randomiser.css:259]: add a newline pls
```suggestion
}

```

File: js/randomiser.js [hunk 2]
```diff
@@ -0,0 +1,378 @@
+
```
Comment (danielleos) [inline js/randomiser.js:1]: remove this line
```suggestion
```

File: js/randomiser.js [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+        if (repeatsToggle === 1) {
+         repeats.push(randomNumber);
+        }
+
+    // IF THERE ARE NO MORE PATTERNS LEFT...
+    } else {
+        document.getElementById("wheel_content").style.display = "none";
+        document.getElementById("end_card").style.display = "block";
+    }
+}
```
Comment (danielleos) [inline js/randomiser.js:378]: add newline
```suggestion
}

```

File: randomiser.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                    <div class="start" onClick="history.go(0);">START AGAIN</div>
+                </div>
+            </div>
+            <div class="settings" id="settings">SETTINGS</div>
+        </div>
+    </div>
+    <script src="js/randomiser.js"></script>
+</body>
+
+</html>
```
Comment (danielleos) [inline randomiser.html:95]: add newline
```suggestion
</html>

```

**Review by danielleos — CHANGES_REQUESTED** [review]

**Comment by ryanleewatts** [comment]
Did it!

**Comment by ryanleewatts** [comment]
Reopening...

**Review by danielleos — APPROVED** [review]

**Pull request merged**